src/lib/components/HoverCard/root.svelte

```diff
@@ -0,0 +1,58 @@
+<script lang='ts' context="module">
+    export type HoverCardRootProps = BaseProps & {
+        /** The controlled open state of the hover card.*/
+        open?: boolean;
+        /** The duration (in ms) from when the mouse enters the trigger until the hover card opens. */
+        openDelay?: number;
+        /** The duration (in ms) from when the mouse leaves the trigger or content until the hover card closes. */
+        closeDelay?: number;
+    };
+
+    export type Context = {
+        open: boolean;
+
+        readonly openDelay: number;
+        readonly closeDelay: number;
+
+        openTimer: NodeJS.Timeout | undefined;
+        closeTimer: NodeJS.Timeout | undefined;
+    };
+
+    const {getContext, setContext} = reactiveContext<Context>();
+    export const getRootContext = getContext;
+</script>
+
+<script lang='ts'>
+	import type { BaseProps } from "$lib/internal/types";
+
+    import {reactiveContext} from '$lib/internal/helpers/reactiveContext';
+
+    import {Popper} from '$lib/internal/components';
+
+    type $$Props = HoverCardRootProps;
+
+    export let open = false;
+    export let openDelay = 750;
+    export let closeDelay = 300;
+    export let use: $$Props['use'] = [];
+
+    const contextStore = setContext({
+        open: [open, v => open = v],
+        openDelay: [openDelay],
+        closeDelay: [closeDelay],
+        openTimer: [undefined],
+        closeTimer: [undefined]
+    });
+
+    $: contextStore.set({
+        open,
+        openDelay,
+        closeDelay,
+        openTimer: undefined,
```

Comment:
This will reset the openTimer and closeTimer whenever any other changes happn. Is this intended? Otherwise, we should use the `update` method.